#!/bin/sh -e
## 309_m68k-pie.dpatch
##
## DP: Description: emulparams/m68kelf.sh (GENERATE_PIE_SCRIPT, COMMONPAGESIZE): Define.
## DP: Upstream status: cvs trunk

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p0 < $0;;
       -unpatch) patch $patch_opts -p0 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
--- ld/emulparams/m68kelf.sh	2007/08/09 11:02:24	1.6
+++ ld/emulparams/m68kelf.sh	2008/03/06 12:29:52	1.7
@@ -2,10 +2,12 @@
 OUTPUT_FORMAT="elf32-m68k"
 TEXT_START_ADDR=0x80000000
 MAXPAGESIZE="CONSTANT (MAXPAGESIZE)"
+COMMONPAGESIZE="CONSTANT (COMMONPAGESIZE)"
 ARCH=m68k
 MACHINE=
 NOP=0x4e754e75
 TEMPLATE_NAME=elf32
 EXTRA_EM_FILE=m68kelf
 GENERATE_SHLIB_SCRIPT=yes
-NO_SMALL_DATA=yes
+GENERATE_PIE_SCRIPT=yes
+NO_SMALL_DATA=yes
\ No newline at end of file
